src/Api/index.js

```diff
@@ -344,15 +344,28 @@ function invariant(condition, errorMsg) {
 }
 
 export function useTrades(chainId, account) {
-  const url =
-    account && account.length > 0
-      ? `${getServerBaseUrl(chainId)}/actions?account=${account}`
-      : `${getServerBaseUrl(chainId)}/actions`;
-  const { data: trades, mutate: updateTrades } = useSWR(url, {
+  let url = getTracerServerUrl(chainId, "/actions");
+  if (account && account.length) {
+    url += `&account=${account}`;
+  }
+
+  const { data, mutate: updateTrades } = useSWR(url, {
```

Comment:
Couple ways this can be done. We should decide on one https://swr.vercel.app/docs/arguments